automated_exports.go

```diff
@@ -0,0 +1,45 @@
+package recurly
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"net/http"
+)
+
+// AutomatedExportsService manages the interactions for automated exports.
+type AutomatedExportsService interface {
+	// Get retrieves export file.
```

Comment:
It would be helpful to add the two List methods so someone can list and feed the name/date directly into the `Get` call.

https://dev.recurly.com/v2.8/docs/list-export-dates
https://dev.recurly.com/v2.8/docs/list-export-files

Reply:
Updated